db/migrate/20150819140659_generate_tokens_for_existing_users.rb

```diff
@@ -0,0 +1,7 @@
+class GenerateTokensForExistingUsers < ActiveRecord::Migration
+  def change
+    User.all.each do |user|
```

Comment:
I agree with you. There's lots of information out there on custom rake tasks. I'll revist this and see if we can get rid of this altogether